what-is-software-development/study-like-a-programmer.md

```diff
@@ -0,0 +1,340 @@
+# How to Study like a Programmer
+
+<iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?pid=c777056e-412b-40f6-999b-ac2501267610&autoplay=false&offerviewer=true&showtitle=true&showbrand=false&start=0&interactivity=all" height="405" width="720" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
+
+## Introduction: Humans vs. Computers
+
+What is the difference between a computer and a programmer?
+
+Computers are strong at
+
+- handling large amounts of data
+- doing large calculations
+- executing algorithms at scale
+- generating things that follow specific rules
+
+Human brains, on the other hand, are strong at
+
+- pattern-finding
+- decision-making
+- designing
+- interpreting
+- jumping between messy layers of abstraction
+- creating from nothing
+
+A key element to thriving as a programmer will be to leverage computers for their strengths, so that you have the resources to work on your own problem-solving skills.
+
+Today, we should learn about what it means to **be** a programmer and **study** like a programmer.
+
+## Learning Goals
+
+- Interpret the role of "software developer" as a creative, human, artistic role
+- Explain that studying as a software developer requires self-care
+- Use these studying tools and techniques to improve question-asking skills
+- Use these studying tools and techniques to improve studying skills
+
+## Consider the Role
+
+There are so many ways to define the role of a software developer and programmer. We're sure you all spent a lot of time considering what it means to you.
+
+Some words that might describe what the role of a software developer is:
+
+- A career
```

Comment:
"Someone who writes stories for machines. These stories almost always build off others - like fan-fictions or sequels."? 😛

(This is definitely true at larger companies/codebases - where you're often trying to stay "canonical" with the rest of the codebase.)